### DISCUSSION TITLE
[V2] How to make the window maximized by default, not full screen

### DISCUSSION THREAD

**Discussion by OneSeven** [wailsapp/wails#911] [Q&A]
How to make the window maximized by default, not full screen.
Can I remove the maximize icon in the toolbar after opening the maximize ？

**Comment by leaanthony** [comment]
You have 2 options:
  - Call the maximise runtime method during startup
  - Open a ticket so we can discuss how we can support it :-)

**Reply by OneSeven** [reply]
v2.0.0-beta.15

Set runtime.WindowMaximise(ctx) in startup,
Can't take effect. Where should I set it?

**Reply by leaanthony** [reply]
Try in DomReady callback

**Reply by OneSeven** [reply]
> Try in DomReady callback

```
func (b *App) domReady(ctx context.Context) {
	runtime.WindowMaximise(ctx)
	// Add your action here
}
```

Still can't work, I tried WindowMaximise in js again. I imagined that it was maximized first, and then it became the default width and height. Is this a BUG?

**Reply by leaanthony** [reply]
Yeah, probably. Please open a ticket with steps to reproduce, and thanks for looking into it 🙏
